iron-ajax.html

```diff
@@ -299,6 +299,14 @@
         value: ''
       },
 
+      /**
+       * Whether the request, response, and error events should bubble to document.
```

Comment:
document -> window

This would normally be super pedantic, but for the error event it's actually relevant.